tests/providers/dbt/cloud/hooks/test_dbt_cloud.py

```diff
@@ -129,6 +129,13 @@ def test_init_hook(self):
         assert hook.auth_type == TokenAuth
         assert hook.method == "POST"
 
+    def test_init_hook_single_tenant(self):
+        hook = DbtCloudHook(tenant="single.tenant")
+        assert hook.dbt_cloud_conn_id == "dbt_cloud_default"
+        assert hook.base_url == "https://single.tenant.getdbt.com/api/v2/accounts/"
+        assert hook.auth_type == TokenAuth
+        assert hook.method == "POST"
```

Comment:
Can you update this test accordingly too?

Reply:
Definitely. I tried my hand at parameterizing this for an additional test connection. Please lmk what you think.